setup.py

```diff
@@ -18,6 +18,7 @@
     packages=['toml'],
     license="MIT",
     long_description=readme_string,
+    python_requires=">=2.6, !=3.0.*, !=3.1.*, !=3.2.*, !=3.2.*",
```

Comment:
3.2 is repeated here.

Reply:
Oops! Thanks for catching. Fixed in the latest revision.